.travis.yml

```diff
@@ -5,12 +5,18 @@ rvm:
   - '2.1'
   - '2.2'
   - '2.3.0'
-  - '2.4.0'
+  - '2.4.1'
```

Comment:
The Mongo gem faces issue of `unexpected argument, expected Proc` - which is due to [this bug](https://bugs.ruby-lang.org/issues/13107) in MRI Ruby 2.4.0 itself.  This has been fixed in v2.4.1. Hence, RSpec tests fail for v2.4.0 and are passing on v2.4.1.

However, I don't like the idea of changing Ruby version to 2.4.1 on Travis just because of one importer. Can the mongo importer be "silenced" / set as "pending" for just Ruby 2.4.0? Or would keeping the 2.4.1 version over 2.4.0 in Travis, be a better alternative?